tracing-appender/README.md

```diff
@@ -0,0 +1,130 @@
+# tracing-appender
+
+Writers for logging events and spans
+
+[![Crates.io][crates-badge]][crates-url]
+[![Documentation][docs-badge]][docs-url]
+[![Documentation (master)][docs-master-badge]][docs-master-url]
+[![MIT licensed][mit-badge]][mit-url]
+[![Build Status][actions-badge]][actions-url]
+[![Discord chat][discord-badge]][discord-url]
+
+[Documentation][docs-url] | [Chat][discord-url]
+
+[crates-badge]: https://img.shields.io/crates/v/tracing-appender.svg
+[crates-url]: https://crates.io/crates/tracing-appender/0.1.0
+[docs-badge]: https://docs.rs/tracing-appender/badge.svg
+[docs-url]: https://docs.rs/tracing-appender/0.1.0
+[docs-master-badge]: https://img.shields.io/badge/docs-master-blue
+[docs-master-url]: https://tracing-rs.netlify.com/tracing-appender
+[mit-badge]: https://img.shields.io/badge/license-MIT-blue.svg
+[mit-url]: LICENSE
+[actions-badge]: https://github.com/tokio-rs/tracing/workflows/CI/badge.svg
+[actions-url]:https://github.com/tokio-rs/tracing/actions?query=workflow%3ACI
+[discord-badge]: https://img.shields.io/discord/500028886025895936?logo=discord&label=discord&logoColor=white
+[discord-url]: https://discord.gg/EeF3cQw
+
+## Overview
+
+[`tracing`][tracing] is a framework for structured, event-based diagnostic information. `tracing-appender` allows
+events and spans to be recorded in a non-blocking manner through a dedicated logging thread. 
+It also provides a [`RollingFileAppender`][file_appender] that can be used with _or_ without the non-blocking writer.
+
+## Usage
+
+Add the following to your `Cargo.toml`:
+```toml
+tracing-appender = "0.1"
+```
+
+This crate can be used in a few ways to record spans/events:
+ - Using a [`RollingFileAppender`][file_appender] to write to a log file. This is a blocking operation.
+ - Using *any* type implementing [`std::io::Write`][write] in a non-blocking fashion.
+ - Using [`NonBlocking`][non_blocking] and [`RollingFileAppender`][file_appender] together to write to write to log files in a non-blocking fashion.
+
+## Rolling File Appender
+
+```rust
+fn main(){
+    let file_appender = tracing_appender::rolling::hourly("/some/directory", "prefix.log");
+}
+```
+This creates an hourly rotating file appender that writes to `/some/directory/prefix.log.YYYY-MM-DD-HH`.
+[`Rotation::DAILY`] and [`Rotation::NEVER`] are the other available options.
+
+The file appender implements [`std::io::Write`][write]. To be used with [`tracing_subscriber::FmtSubscriber`][fmt_subscriber], 
+it must be combined with a [`MakeWriter`][make_writer] implementation to be able to record tracing spans/event.
+
+The [rolling] module's documentation provides more detail on how to use this file appender.
+
+## Non-Blocking Writer
+The example below demonstrates the construction of a `non_blocking` writer with `std::io::stdout()`, which implements [`MakeWriter`][[make_writer].
```

Comment:
This example no longer uses `io::stdout`, can we update this?

Reply:
Sure